lerna.json

```diff
@@ -18,7 +18,6 @@
   ],
   "version": "2.0.0-rc.1",
   "npmClient": "yarn",
-  "useWorkspaces": "true",
```

Comment:
@jenniferlianne Why was it necessary to diable this flag?

Reply:
The 'npx lerna' command under ci invokes the lerna 8.1 installed on the runner, not lerna lite.  lerna 8.1 does not support useWorkspaces, which is deprecated.